### DISCUSSION TITLE
What is hotkey to open Project Manager Dialog?

### DISCUSSION THREAD

**Discussion by knightmare110** [alefragnani/vscode-project-manager#750] [Q&A] [answered]
I installed Project Manager and set the root folder.
But not sure how I can open up the modal by using keyboard to list projects in root folder.

Thanks.

**Comment by alefragnani** [comment, marked as answer]
Hi @knightmare110 ,

The extension has only one keybinding defined, to the `Project Manager: List Projects to Open` command. If you are on a Mac, it will be <kbd>Cmd</kbd>+<kbd>Alt</kbd>+<kbd>P</kbd>. In Windows or Linux, it will be <kbd>Ctrl</kbd>+<kbd>Alt</kbd>+<kbd>P</kbd>.

About the _root folder_ you are refering to. Do you mean the `projectManager.git.baseFolders` setting, like below?

```json
    "projectManager.git.baseFolders": [
        "~/Documents/GitHub",
        "/Volumes/ExternalDisk/sources"
    ],
```

Or did you edited `projects.json` and defined the `rootFolder` property of one of the projects, like:

```
[
	{
		"name": "James",
		"rootPath": "c:\\sources\\delphi-james",
		"paths": [],
		"tags": [
			"Personal"
		],
		"enabled": true
	}
]
``` 

Hope this helps
